### DISCUSSION TITLE
[v1.9.2 hw2 gamecube] Xbox Series X controller losing L/R trigger upon system wake

### DISCUSSION THREAD

**Discussion by jspector22** [darthcloud/BlueRetro#994] [General]
I've configured and mapped my XboxSX controller to the gamecube successfully. When I CB SYS Power Off and reconnect my controller, the system wakes as expected but without either my L or R trigger working. Its usually just one trigger is unresponsive. This is resolved by disconnecting and reconnecting the controller, then L/R function like normal.

**Comment by darthcloud** [comment]
Read this note abour axis calibration:
https://github.com/darthcloud/BlueRetro/wiki/Controller-pairing-guide#ii---blueretro-joystick-calibration

Most likely the issue
